test/integration/client-navigation/test/index.test.js

```diff
@@ -479,6 +479,14 @@ describe('Client Navigation', () => {
             .eval('window.pageYOffset')
 
           expect(scrollPositionAfterEmptyHash).toBe(0)
+
+          // Scrolls back to top when clicking link with href `#top`.
+          const scrollPositionAfterTopHash = await browser
+            .elementByCss('#via-top-hash')
+            .click()
+            .eval('window.pageYOffset')
+
+          expect(scrollPositionAfterTopHash).toBe(0)
```

Comment:
The position of the scroll is already at the top due to the code above this. Could you scroll down to another element and then scroll to the top to make sure it works 🙏

Reply:
done